examples/models/llama/export_llama_lib.py

```diff
@@ -128,6 +132,12 @@ def build_args_parser() -> argparse.ArgumentParser:
     # parser.add_argument(
     #     "-q", "--quantized_ckpt", default=None, help="quantized checkpoint file"
     # )
+    parser.add_argument(
+        "--model",
+        default="llama3",
+        choices=EXECUTORCH_DEFINED_MODELS + TORCHTUNE_DEFINED_MODELS,
+        help="The Lllama model to export. llama2, llama3, llama3_1, llama3_2 share the same architecture, so they are technically interchangeable given you provide the checkpoint file for the desired version.",
+    )
```

Comment:
I'm wondering what is the motivation for adding this command line option? All llama models share the same model architecture and code. The `modelName` in the codebase does not really do any thing. Everything should continue to work if we hardcode a name for it or remove it. On the other hand, if we add one more command line option, it adds more work for people who need to use `export_llama`, `eval_llama` commands by making them have more options to add.

I feel we should change `class Llama2Model` to `class LlamaModel` instead.